microservices/first/main.py

```diff
@@ -1,27 +1,28 @@
 # Endpoint documentation: https://github.com/jakejack13/SPEED/blob/microservices/first_api_doc.md
-import json
 from utils import DBManager
 import utils
 
 from flask import Flask, request, jsonify
+import flask
```

Comment:
double import of flask?

Reply:
This way, I can use `flask.Response` in a type annotation without MyPy complaining that I haven't used `Response`